src/internal/operators/observeOn.ts

```diff
@@ -49,19 +50,58 @@ import { OperatorSubscriber } from './OperatorSubscriber';
  *
  * @see {@link delay}
  *
- * @param {SchedulerLike} scheduler Scheduler that will be used to reschedule notifications from source Observable.
- * @param {number} [delay] Number of milliseconds that states with what delay every notification should be rescheduled.
+ * @param scheduler Scheduler that will be used to reschedule notifications from source Observable.
+ * @param delay Number of milliseconds that states with what delay every notification should be rescheduled.
  * @return A function that returns an Observable that emits the same
  * notifications as the source Observable, but with provided scheduler.
  */
-export function observeOn<T>(scheduler: SchedulerLike, delay: number = 0): MonoTypeOperatorFunction<T> {
+export function observeOn<T>(scheduler: SchedulerLike, delay = 0): MonoTypeOperatorFunction<T> {
   return operate((source, subscriber) => {
+    /**
```

Comment:
@kwonoj please review, I've changed the approach just a little bit, and I've added detailed comments.

Reply:
🚢